includes/Core/Assets/Assets.php

```diff
@@ -817,14 +787,15 @@ private function get_external_assets() {
 							'document.contains'                                      => $base_url . 'js/externals/wp-polyfill-node-contains.js', // phpcs:ignore WordPress.Arrays.MultipleStatementAlignment
 							'window.FormData && window.FormData.prototype.keys'      => $base_url . 'js/externals/wp-polyfill-formdata.js', // phpcs:ignore WordPress.Arrays.MultipleStatementAlignment
 							'Element.prototype.matches && Element.prototype.closest' => $base_url . 'js/externals/wp-polyfill-element-closest.js', // phpcs:ignore WordPress.Arrays.MultipleStatementAlignment
+							'typeof URL === \'function\''                            => $base_url . 'js/externals/wp-polyfill-url.js', // phpcs:ignore WordPress.Arrays.MultipleStatementAlignment
 						);
-						$polyfill_scripts = '/*googlesitekit*/';
+						$polyfill_scripts = '/*googlesitekit*/' . "\n";
 						foreach ( $inline_polyfill_tests as $test => $script ) { // phpcs:ignore Generic.WhiteSpace.ScopeIndent.IncorrectExact
 							$polyfill_scripts .= (
 								'( ' . $test . ' ) || ' .
 								'document.write( \'<script src="' . // phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript
-								$script .
-								'"></scr\' + \'ipt>\' );'
+								$script . '?ver=' . GOOGLESITEKIT_VERSION .
+								'"></scr\' + \'ipt>\' );' . "\n"
```

Comment:
What's with the split `</script>` tag here? If it's just a PHPCS security warning we could silence it instead 😄

Reply:
Because this is a `document.write` call, not splitting up `</script>` would actually close the enclosing `script` tag rather than _writing_ it.

Reply:
Ah, whoops, right. Thanks 😅